app/controllers/issue_templates_settings_controller.rb

```diff
@@ -1,26 +1,25 @@
 class IssueTemplatesSettingsController < ApplicationController
   unloadable
   before_filter :find_project, :find_user
-  before_filter :authorize, :except => [ :show_help, :preview ]
+  before_filter :authorize, except: [ :show_help, :preview ]
 
   def edit
     if params[:settings] != nil
       @issue_templates_setting = IssueTemplateSetting.find_or_create(@project.id)
       attribute = params[:settings]
-      @issue_templates_setting.update_attributes(:enabled => attribute[:enabled],
-                                                 :help_message => attribute[:help_message],
-                                                 :inherit_templates => attribute[:inherit_templates],
-                                                 :should_replaced => attribute[:should_replaced])
+      @issue_templates_setting.update_attributes(enabled: attribute[:enabled],
+                                                 help_message: attribute[:help_message],
+                                                 inherit_templates: attribute[:inherit_templates],
+                                                 should_replaced: attribute[:should_replaced])
 
       flash[:notice] = l(:notice_successful_update)
-      redirect_to :controller => 'projects', 
-        :action => "settings", :id => @project, :tab => 'issue_templates'
+      redirect_to controller: 'projects', action: "settings", id: @project, tab: 'issue_templates'

```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Line is too long. [98/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)